tests/e2e/multus/Vagrantfile

```diff
@@ -0,0 +1,97 @@
+ENV['VAGRANT_NO_PARALLEL'] = 'no'
+NODE_ROLES = (ENV['E2E_NODE_ROLES'] || ["server-0", "agent-0" ])
+NODE_BOXES = (ENV['E2E_NODE_BOXES'] || ['generic/ubuntu2004', 'generic/ubuntu2004'])
+GITHUB_BRANCH = (ENV['E2E_GITHUB_BRANCH'] || "master")
+RELEASE_VERSION = (ENV['E2E_RELEASE_VERSION'] || "")
+NODE_CPUS = (ENV['E2E_NODE_CPUS'] || 2).to_i
+NODE_MEMORY = (ENV['E2E_NODE_MEMORY'] || 3072).to_i
+CNI = (ENV['E2E_CNI'] || "multus,canal") # canal, cilium (with cni.excludsive: false in the chart values) and calico supported
```

Comment:
excludsive?? typo. Also I would remove the ability to override the cni via ENV, and just hardcode the value.

Reply:
Typo indeed. I thought QA was running those tests and changing the CNI plugineach time to see if things still worked